product_template_tags/models/product_template_tag.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplateTag(models.Model):
+
+    _name = 'product.template.tag'
+    _description = 'Product Tag'
+
+    name = fields.Char(string="Name", required=True, translate=True)
+    color = fields.Integer(string="Color Index")
+    product_tmpl_ids = fields.Many2many(
+        comodel_name='product.template', string="Products",
+        relation='product_template_product_tag_rel',
+        column1='tag_id', column2='product_tmpl_id')
+    products_count = fields.Integer(
+        string="# of Products", compute='_compute_products_count', store=True)
+
```

Comment:
could you add a company_id field (optional) ? with an according ir.rule. (access OK if company_id = current company or False). (and displayed only if user is part of multicompany)
it will be usefull in all multi company context.

Reply:
Indeed, this could be a nice feature, I will add it.

Reply:
Multi company feature added!